src/scanner/images/index.ts

```diff
@@ -119,10 +119,14 @@ export async function scanImages(
   for (const { imageName, fileSystemPath, imageWithDigest } of images) {
     try {
       const archivePath = `docker-archive:${fileSystemPath}`;
+      const imageNamesOverride: string[] = !!imageWithDigest
```

Comment:
`imageWithDigest` should be `string`? Why does it need `!!`?

Reply:
`!!` turns it into a boolean to use in the ternary condition so if there is `imageWithDigest`, add it to the array with `imageName`, if not then the array contains just `imageName`

Reply:
but if `imageWithDigest` is a valid string, on it is own in the ternary would return truthy value anyway, if the `imageWithDigest` is an empty string, then it would be falsy anyway, right?